assets/scss/global/_settings.scss

```diff
@@ -35,10 +35,11 @@
 //  30. Table
 //  31. Tabs
 //  32. Thumbnail
-//  33. Tooltip
-//  34. Top Bar
+//  33. Title Bar
+//  34. Tooltip
+//  35. Top Bar
 
-@import '../../components/foundation-sites/scss/util/util';
+@import 'util/util';
```

Comment:
@colin-marshall : Not sure about this one? The _settings.scss file is located in our assets/scss/global folder. The correct path for the import statement of _util.scss would be ```'../../components/foundation-sites/scss/util/util';``` . Or am I missing anything?